libraries/cyclestreets-view/src/main/java/net/cyclestreets/tiles/TileSource.java

```diff
@@ -112,32 +112,23 @@ public static ITileSource createStandardTileSource(final String name, final Stri
   public static ITileSource createStandardTileSource(final String name,
                                                      final ResourceProxy.string aResourceId,
                                                      final String... baseUrls) {
-    return createXYTileSource(name, aResourceId, 256, ".png", baseUrls);
+    return createXYTileSource(name, 256, ".png", baseUrls);
   } // createStandardTileSource
 
   public static ITileSource createDensityAwareTileSource(final Context context,
                                                          final String name,
                                                          final String... baseUrls) {
-    return createDensityAwareTileSource(context, name, ResourceProxy.string.unknown, baseUrls);
-  } // createDensityAwareTileSource
-
-  public static ITileSource createDensityAwareTileSource(final Context context,
```

Comment:
I believe so - see my [comment](https://github.com/cyclestreets/android/issues/133#issuecomment-232309900) in the related issue - but that's one of the things I want to validate through actual testing.